microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -269,4 +269,13 @@ Issued: 2024-04-08
 
 {product-title} release 4.15.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1670[RHBA-2024:1670] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1668[RHSA-2024:1668] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deploy_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-15-9-dp"]
+=== RHBA-2024:1772 - {microshift-short} 4.15.9 bug fix and security update advisory
+
+Issued: 2024-04-16
```

Comment:
OK that date is in the future?

Reply:
This PR  will be merged only after the RHBA is shipped live on 16th April. So should be good.